### PR TITLE
Add respondent management session schema

### PR DESCRIPTION
**Pull request by dhilton**
__What__
This PR adds the claim definition for a respondent management system creating a
new session on eq for a user to coimplete a survey.

__How to review__

1. Look at the documentation added in this PR
2. Ensure it matches the understanding you have of the process

__Who can review__

Anyone but @dhilton

### CODE REVIEW

File: docs/rm-eq.rst [hunk 1]
```diff
@@ -1,9 +1,64 @@
...
-Session timeouts
+
+
+An example JSON claim
+=====================
+
+.. code-block:: javascript
+
+  {
+    "user_id": "64389274239",
```
Comment (collisdigital) [inline docs/rm-eq.rst]: Add type/version/origin as per sdx?

File: docs/rm-eq.rst [hunk 2]
```diff
@@ -1,9 +1,64 @@
...
+    "ru_ref": "7897897",
+    "ru_name": "",
+    "eq_id": "678",
+    "collection_exercise_sid": "789",
+    "period_id": "",
+    "period_str": "",
+    "ref_p_start_date": "",
+    "ref_p_end_date": "",
+    "form_type": ""
+  }
```
Comment (warrenbailey) [inline docs/rm-eq.rst]: I think it's also worth documenting the IAT and EXP claims for the JWT

File: docs/rm-eq.rst [hunk 3]
```diff
@@ -1,9 +1,64 @@
...
+  period_id
+    A numerical reference to either a month or quarter time period
+  period_str
+    A display name for the period referenced above
+  ref_p_start_date
+    A start date for the relevant reference period
+  ref_p_end_date
+    An end date for the relevant reference period
+  form_type
+    The particular form_type for a responding unit
```
Comment (warrenbailey) [inline docs/rm-eq.rst]: Are these all Mandatory? Should we be rejecting the JWT if it doesn't include them?

File: docs/rm-eq.rst (hunk 2 above)
Comment (dhilton) [inline docs/rm-eq.rst]: Let's do the transport as a separate PR - totally agree though.

File: docs/rm-eq.rst (hunk 3 above)
Comment (dhilton) [inline docs/rm-eq.rst]: All our mandatory right now - will amend the docs to state this explicitly.

**Comment by warrenbailey** [comment]
LGTM :+1:
